integration_tests/src/main/python/date_time_test.py

```diff
@@ -468,12 +468,32 @@ def test_gettimestamp_ansi_exception():
 supported_date_formats = ['yyyy-MM-dd', 'yyyy-MM', 'yyyy/MM/dd', 'yyyy/MM', 'dd/MM/yyyy',
                           'MM-dd', 'MM/dd', 'dd-MM', 'dd/MM']
 @pytest.mark.parametrize('date_format', supported_date_formats, ids=idfn)
-@pytest.mark.parametrize('data_gen', date_n_time_gens, ids=idfn)
-@allow_non_gpu(*non_utc_allow)
-def test_date_format(data_gen, date_format):
+@pytest.mark.parametrize('data_gen', [date_gen], ids=idfn)
+@allow_non_gpu('ProjectExec')
```

Comment:
need to check whether it's supported timezone?

Reply:
date_format(date) will introduce cast(date as timestamp) which is not supported in non-UTC now. 
After we support cast(date as timestamp), then we will update this case.